src/client/scripts/esm/chess/logic/movepiece.js

```diff
@@ -55,11 +55,12 @@ import game from '../../game/chess/game.js';
  * - `doGameOverChecks`: Whether to perform game-over checks, such as checkmate or other win conditions.
  * - `concludeGameIfOver`: If true, and `doGameOverChecks` is true, then if this move ends the game, we will not stop the clocks, darken the board, display who won, or play a sound effect.
  * - `animate`: Whether to animate this move.
+ * - `animateSecondary`: Animate the pieces affected by the move without the piece that made the move. Used after dragging the king to castle.
```

Comment:
I like the description you added! It's just unclear if it overwrites `animate` or if that overwrites `animateSecondary`